Auth/server/controllers/authController.js

```diff
@@ -1,28 +1,21 @@
-const authService = require('../services/AuthService.js');
 const { getJwtToken } = require('../helpers/jwt.js');
 const jwt = require('jsonwebtoken');
 const env = require('../../loadEnvironment.js');
 
 // Define a controller function for handling login requests
-const login = async (req, res) => {
+const generate = async (req, res) => {
   try {
-    const { email, password } = req.body;
-    const user = await authService.loginUser(email, password);
-    const jwtToken = getJwtToken(user);
-    res.json({ message: 'User logged in successfully', token: jwtToken });
-  } catch (err) {
-    res.status(err?.status || 400).json({ error: err?.message || err });
-  }
-};
+    const { email, isMaintainer } = req.body;
 
-// Define a controller function for handling signup requests
-const signup = async (req, res) => {
-  try {
-    const { email, password, confirmPassword } = req.body;
-    await authService.createUser(email, password, confirmPassword);
-    res.json({ message: 'User registered successfully' });
+    tokenData = {
+      email: email,
+      isMaintainer: isMaintainer,
+    }
+    
+    const jwtToken = getJwtToken(tokenData);
```

Comment:
Might wanna put this in services/AuthService.js because I think controller job is just to extract the fields from the request body.

Reply:
I'm not entirely sure how to let AuthService handle this